backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/preloader/Preloader.java

```diff
@@ -132,25 +134,25 @@ public void onSuccess (String result) {
 				Array<Asset> assets = new Array<Asset>(lines.length);
 				for (String line : lines) {
 					String[] tokens = line.split(":");
-					if (tokens.length != 4) {
+					if (tokens.length != 5) {
 						throw new GdxRuntimeException("Invalid assets description file.");
 					}
 					AssetType type = AssetType.Text;
 					if (tokens[0].equals("i")) type = AssetType.Image;
 					if (tokens[0].equals("b")) type = AssetType.Binary;
 					if (tokens[0].equals("a")) type = AssetType.Audio;
 					if (tokens[0].equals("d")) type = AssetType.Directory;
-					long size = Long.parseLong(tokens[2]);
+					long size = Long.parseLong(tokens[3]);
```

Comment:
(The problem was already here before this PR, I just note it here.) The token index numbers should be covered in constants named, so it is better understandable what each index is used for. Reference comment to PreloaderBundleGenerator should be added to these constants so it is better understandable what's happening here.